joss/paper.md

```diff
@@ -166,14 +168,14 @@ The `fitgrid` documentation is available online:
 
   ![Downloadable Examples Gallery](examples_gallery.png)
```

Comment:
same as above: Maybe slightly expand the figure caption.

Reply:
Done